src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -43,7 +43,10 @@ public ArrayList<Integer> getFavouriteNumbers() {
         TODO: 1. Create a method named getSecondNumber that returns a whole number. It must return the
            second number contained in the list that is returned from getFavouriteNumbers
      */
-
+    public int getSecondNumber(){
+        int secondNumber = this.getFavouriteNumbers().get(1);
+        return secondNumber;
+    }
```

Comment:
Whenever setting a var then returning it, you can think if it's cleaner to just return the value, eg:

```
public int getSecondNumber(){
        return this.getFavouriteNumbers().get(1);
   }
```